src/System.CommandLine.Subsystems.Tests/Constants.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Reflection;
+
+namespace System.CommandLine.Subsystems.Tests
+{
+    internal class Constants
+    {
+
+        internal static readonly string? version = (Assembly.GetEntryAssembly() ?? Assembly.GetExecutingAssembly())
+                                                ?.GetCustomAttribute<AssemblyInformationalVersionAttribute>()
+                                                ?.InformationalVersion;
+
+        internal static readonly string newLine = Environment.NewLine;
```

Comment:
Nit: We have typically named these in PascalCase.
```suggestion
        internal static readonly string? Version = (Assembly.GetEntryAssembly() ?? Assembly.GetExecutingAssembly())
                                                ?.GetCustomAttribute<AssemblyInformationalVersionAttribute>()
                                                ?.InformationalVersion;

        internal static readonly string NewLine = Environment.NewLine;
```